posix.mak

```diff
@@ -532,7 +532,7 @@ $(DSCANNER_DIR)/dsc: | $(DSCANNER_DIR) $(DMD) $(LIB)
 	# debug build is faster, but disable 'missing import' messages (missing core from druntime)
 	sed 's/dparse_verbose/StdLoggerDisableWarning/' $(DSCANNER_DIR)/makefile > $(DSCANNER_DIR)/dscanner_makefile_tmp
 	mv $(DSCANNER_DIR)/dscanner_makefile_tmp $(DSCANNER_DIR)/makefile
-	DC=$(abspath $(DMD)) DFLAGS="$(DFLAGS) -defaultlib=$(LIB)" $(MAKE) -C $(DSCANNER_DIR) githash debug
+	DC=$(abspath $(DMD)) DFLAGS="$(DFLAGS) -release -defaultlib=$(LIB)" $(MAKE) -C $(DSCANNER_DIR) githash debug
```

Comment:
i think that you must remove `debug after `githash`. The DMD flags are then set in the makefile.

Reply:
and remove this `-release` too.

Reply:
And it will be named `dscanner` so the `dsc` have to be changed

Reply:
renaming to `dscanner` doesn't work.  

`make: generated/dscanner-39496ede1a2c00674c4e04b6513be7cc9aee6cef/dscanner: Command not found`